properties.py

```diff
@@ -264,6 +264,16 @@ def update_type(self, context):
         name="AOV name",
         description="Name of the built in AOV")
 
+    denoise_aov = BoolProperty(
+        name="Format AOV for denoising",
+        description="If checked this pass will be export as an individual file and properly formatted for use by the denoise utility",
+        default=False)
+
+    exclude_from_multi = BoolProperty(
```

Comment:
No I mean "Why would a user want to make an aov but not put it in the multilayer?  Not saying it hurst anything just not sure it's needed.